homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -142,6 +150,7 @@
         "method": "async_clean_zone",
         "schema": SERVICE_SCHEMA_CLEAN_ZONE,
     },
+    SERVICE_GOTO: {"method": "async_goto", "schema": SERVICE_SCHEMA_GOTO},
```

Comment:
Do you mean using `platform.async_register_entity_service` instead of `hass.services.async_register`? I can't find that it is used anywhere in the xiaomi_miio/remote.py. Can you point to an exact place for an example?

Reply:
Never mind, found the code in xiaomi_miio remote now. I'll look into it.